.github/workflows/build_and_test.yml

```diff
@@ -2,24 +2,35 @@ name: Build and test
 
 on:
   pull_request:
+    branches:
+      - ros2
   push:
     branches:
       - ros2
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
I suppose these are old changes to CI and could now be removed from this PR?

Reply:
These changes are no longer in the latest commit.